api.web.yaml

```diff
@@ -1311,11 +1311,45 @@ components:
         study_id:
           type: string
           description: Unique identifier of the study to which the project belongs
+        is_draft:
+          type: boolean
+          description: Whether this project should be saved as a draft (true) or created as active (false)
+          default: false
```

Comment:
How will you set `is_draft` to false for submitting a draft, is it a forthcoming PUT variant of the endpoint?